lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/RandomUtils.java

```diff
@@ -0,0 +1,51 @@
+package org.enso.desktopenvironment;
+
+import java.util.Random;
+
+final class RandomUtils {
+
+  private static final char[] ALPHANUMERIC =
+      new char[] {
+        '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h',
+        'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
+        'A', 'B', 'C', 'D', 'E', 'F', 'J', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R',
+        'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'
+      };
+
+  private RandomUtils() {}
+
+  /** Delay the {@link Random} instance initialization when building the native image. */
+  private static final class LazyRandom {
+
+    private static Random instance = null;
+
+    private LazyRandom() {}
+
+    public static Random getInstance() {
```

Comment:
If you really want to...

> To initialize it once

... then this method is missing `synchronized` modifier.